webpack.config.ts

```diff
@@ -412,9 +409,7 @@ const appConfig: webpack.Configuration = {
 
   resolveLoader: {
     alias: {
-      'type-loader': STORYBOOK_TYPES
-        ? path.resolve(__dirname, 'static/app/stories/type-loader.ts')
-        : path.resolve(__dirname, 'static/app/stories/noop-type-loader.ts'),
+      'type-loader': path.resolve(__dirname, 'static/app/stories/type-loader.ts'),
```

Comment:
I moved the condition into `type-loader.ts` itself, because otherwise, `knip` will see either one of the files as unused, depending on if we have set the env var or not.